py/DREAM/Settings/OtherQuantities.py

```diff
@@ -28,9 +28,12 @@ class OtherQuantities:
         'fluid/Eceff',
         'fluid/GammaAva', 'fluid/GammaAvaAlt',
         'fluid/pCrit',
+        'fluid/GammaCompton',
+        'fluid/GammaDreicer',
```

Comment:
I would prefer if we could keep this list in alphabetical order (I realize that the lack of order most likely predates your changes Oskar, sorry about that :P)